images/linux/scripts/installers/r.sh

```diff
@@ -4,7 +4,10 @@
 source $HELPER_SCRIPTS/document.sh
 
 # install R
-sudo apt-get install -y r-base
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/'
```

Comment:
why is it pointed to bionic only? What about ubuntu 16&20?

Reply:
Done.